workshop/content/exercises/D-01-content.md

```diff
@@ -15,6 +13,13 @@ Triggering `mvn clean build` on a git commit would do all of these tasks. Howeve
 
 This can be done in different ways. We're first going to use `mvn clean deploy` to test and ensure the app builds. Then we are choosing to copy (via a forced git push) the code into a separate repository, `cat-service-release`. Presumably this release repository could have more limited write access than cat-service (e.g. only pipeline system account can write to it). And then the container will be built. We will see the container building part in the next section.
 
+Click on the following GitHub link and use the GitHub UI to fork the `cat-service-release` repo like in the begining of the workshop.
```

Comment:
*beginning

Reply:
Honestly I would make this a "Click on the upper right blah blah and create a new repository. There is no reason to clone this an inherit what is there. I think the concept is more clear if it is new, an it will be easier to see when it goes from having nothing to having something.

Reply:
now link makes new repo immediately instead of forking